tests/dryrun/util.go

```diff
@@ -0,0 +1,173 @@
+package dryrun
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	k0sv1beta1 "github.com/k0sproject/k0s/pkg/apis/k0s/v1beta1"
+	"github.com/replicatedhq/embedded-cluster/pkg/cmd"
+	"github.com/replicatedhq/embedded-cluster/pkg/defaults"
+	dryruntypes "github.com/replicatedhq/embedded-cluster/pkg/dryrun/types"
+	"github.com/replicatedhq/embedded-cluster/pkg/helm"
+	"github.com/replicatedhq/embedded-cluster/pkg/release"
+	troubleshootv1beta2 "github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/yaml"
+)
+
+//go:embed assets/install-release.yaml
+var releaseData string
+
+func dryrunInstall(t *testing.T, args ...string) dryruntypes.DryRun {
```

Comment:
it would be nice to export functions that are intended to be used in testing to make this package more self documenting including this and the assert functions

Reply:
this is only intended to be used for dryrun tests though